app/controller/command/commands/team.py

```diff
@@ -610,3 +617,36 @@ def refresh_helper(self, user_id) -> ResponseTuple:
             f"{num_deleted} deleted. Wonderful."
         ret = {'attachments': modified, 'text': status}
         return ret, 200
+
+    def refresh_all_team(self):
+        """
+        Refresh the 'all' team.
```

Comment:
For future reference perhaps?

```suggestion
        Refresh the 'all' team - this team is used to track all members of UBC Launch Pad.
        See https://github.com/orgs/ubclaunchpad/teams/all
```